Symfony/CS/Tests/Fixer/Symfony/NoEmptyCommentFixerTest.php

```diff
@@ -139,7 +139,7 @@ public function provideFixCases()
             ),
             array(
                 '<?php
-
+                
```

Comment:
This test makes sure the fixer doesn't touch an empty php file it seems.
Since I've `trim all white space` on in PHPStorm it got removed on the save of the namespace change, I've reverted this change.